spec/spec_helper.rb

```diff
@@ -24,16 +24,18 @@ module Shell
   IRB = nil unless defined? IRB
 end
 
-$LOAD_PATH.unshift File.expand_path("../..", __FILE__)
-
-$LOAD_PATH.unshift File.expand_path("../../chef-config/lib", __FILE__)
-$LOAD_PATH.unshift File.expand_path("../../chef-utils/lib", __FILE__)
+# Ruby 1.9 Compat
+$:.unshift File.expand_path("../..", __FILE__)
```

Comment:
try this:

```
$LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
$LOAD_PATH.unshift File.expand_path("../../chef-config/lib", __FILE__)
$LOAD_PATH.unshift File.expand_path("../../chef-utils/lib", __FILE__)
$LOAD_PATH.unshift File.dirname(__FILE__)
```

Reply:
Tests fail with this

Reply:
this should fix that:

```
$LOAD_PATH.unshift File.expand_path("../..", __FILE__)
$LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
$LOAD_PATH.unshift File.expand_path("../../chef-config/lib", __FILE__)
$LOAD_PATH.unshift File.expand_path("../../chef-utils/lib", __FILE__)
$LOAD_PATH.unshift File.dirname(__FILE__)
```

Reply:
[WARN] This is an internal command used by the ChefDK development team. If you are a ChefDK user, please do not run it.
Running verification for component 'chef-client'
......



bundler: failed to load command: /opt/chefdk/embedded/bin/rspec (/opt/chefdk/embedded/bin/rspec)

Bundler::GemNotFound: Could not find rake-13.0.1 in any of the sources
  /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/spec_set.rb:91:in `block in materialize'
  /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/spec_set.rb:85:in `map!'
  /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/spec_set.rb:85:in `materialize'
  /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/definition.rb:170:in `specs'
  /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/definition.rb:237:in `specs_for'
  /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/definition.rb:226:in `requested_specs'
  /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/runtime.rb:108:in `block in definition_method'
  /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/runtime.rb:20:in `setup'
  /opt/chefdk/embedded/lib/ruby/2.6.0/bundler.rb:107:in `setup'
  /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/setup.rb:20:in `<top (required)>'
  /opt/chefdk/embedded/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
  /opt/chefdk/embedded/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'


---------------------------------------------
Verification of component 'chef-client' failed.

Reply:
Alright after fixing some issues with knife-windows this is confirmed working